### PR TITLE
Clare/issue#35/button ripple from click

### PR DESCRIPTION
**Pull request by blaise17**

### CODE REVIEW

File: src/app/obf-button/obf-button.component.html [hunk 1]
```diff
@@ -1,5 +1,17 @@
-<div [class]="'buttonDiv' + (clickHandler ? ' ripple' : '')" [style.background-color]="butt.backgroundColor" [style.border-color]="butt.borderColor"
-                      (click)="clickHandler && clickHandler(butt)">
+<div mat-ripple class="buttonDiv ripple" [style.background-color]="butt.backgroundColor" [style.border-color]="butt.borderColor"
```
Comment (psi77) [inline src/app/obf-button/obf-button.component.html]: You could use
```
<div mat-ripple [matRippleDisable]="!clickHandler" ...>
```
And then you don't need the content template and *ngIf

Comment (psi77) [reply src/app/obf-button/obf-button.component.html]: I think

Comment (psi77) [reply src/app/obf-button/obf-button.component.html]: I just tried it, yeah you can!

```
<div appPageTurner [appPageTurnerCondition]="butt.loadBoardAction"
     mat-ripple [matRippleDisabled]="!clickHandler"
     class="buttonDiv ripple" [style.background-color]="butt.backgroundColor" [style.border-color]="butt.borderColor"
     #buttonDiv (click)="clickHandler && clickHandler(butt)">

  <div class="buttonImageDiv" *ngIf="butt.getImage() as image">
    <div *ngIf="image.isSVG(); else normalImage" [innerHTML]="image.getSource() | safe">SVG Image</div>
    <ng-template #normalImage>
      <img class="buttonImage" [src]="image.getSource()" *ngIf="image.getSource()" />
    </ng-template>
  </div>
  <div class="buttonLabel">
    {{butt.label}}
  </div>

</div>
```

Comment (blaise17) [reply src/app/obf-button/obf-button.component.html]: You absolute BEAUTY!

**Pull request merged**